### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by aladdin91**
this is my PR for js1 week2 homework

### CODE REVIEW

File: javascript/javascript1/week2/Candy-helper.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+let boughtCandyPrices = [];
+const yourLimit = Math.floor(Math.random() * 100);
+let youBalance = 0;
+
+function canBuyMoreCandy() {
+    for (i = 0; i < boughtCandyPrices.length; i++) {
+        youBalance += boughtCandyPrices[i]
+    }
+
+    if (youBalance < yourLimit) {
```
Comment (naresh699) [inline javascript/javascript1/week2/Candy-helper.js:16]: default is true so you can write condition only for false

File: javascript/javascript1/week2/Candy-helper.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        return true
+    } else {
+        return false
+    }
+
+}
+
+function addCandy(candyType, weight) {
+    if (canBuyMoreCandy) {
```
Comment (naresh699) [inline javascript/javascript1/week2/Candy-helper.js:25]: if we have multiple levels of if, else condition, better to go for switch condition

File: javascript/javascript1/week2/Student-manager.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const class07Students = [];
```
Comment (naresh699) [inline javascript/javascript1/week2/Student-manager.js:1]: avoid numbers in names

**Review by naresh699 — COMMENTED** [review]
good work alaa.

File: javascript/javascript1/week2/Candy-helper.js (hunk 2 above)
Comment (aladdin91) [reply javascript/javascript1/week2/Candy-helper.js:25]: yah i will rewrite it again

**Pull request merged**